tgui/packages/tgui/interfaces/AntagInfoRules.tsx

```diff
@@ -0,0 +1,272 @@
+import { useBackend } from '../backend';
+import { Stack } from '../components';
+import { Objective } from './common/Objectives';
+
+type Info = {
+  antag_name: string;
+  objectives: Objective[];
+};
+
+export const Rules = (props, context) => {
+  const { data } = useBackend<Info>(context);
+  const { antag_name } = data;
+  switch (antag_name) {
+    case 'Abductor Agent' || 'Abductor Scientist' || 'Abductor Solo':
```

Comment:
I was hoping we'd have a cleaner way of doing this, but honestly it's good enough for now.